hnn_core/extracellular.py

```diff
@@ -504,16 +509,14 @@ def _build(self, cvode=None, include_celltypes='all'):
             else:
                 # for testing, make a matrix of ones
                 self._nrn_r_transfer.setrow(row,
-                                            h.Vector(self.n_total_segments,
-                                                     1.))
+                                            h.Vector(n_total_segments, 1.))
 
         # record time for each array
         self._nrn_times = h.Vector().record(h._ref_t)
 
         # contributions of all segments on this rank to total calculated
         # potential at electrode (_PC.allreduce called in _simulate_dipole)
         self._nrn_voltages = h.Vector()
-        self._nrn_imem_vec = h.Vector(self.n_total_segments)
```

Comment:
Note that I took the liberty of deleting this line, as it is a duplicate of L482.